backend/src/main/scala/cromwell/backend/BackendCacheHitCopyingActor.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendJobExecutionActor.{AbortedResponse, BackendJobExecutionResponse, ExecuteJobCommand, FailedNonRetryableResponse}
+import cromwell.backend.BackendLifecycleActor._
+
+import scala.concurrent.Future
+
+
+trait BackendCacheHitCopyingActor extends Actor with ActorLogging with BackendJobLifecycleActor {
+
+  def copyCachedOutputs: Future[BackendJobExecutionResponse]
+
+  def receive: Receive = LoggingReceive {
+    case ExecuteJobCommand =>
+      def copyThenShutdown = copyCachedOutputs andThen { case _ => context stop self }
+      performActionThenRespond(copyThenShutdown, onFailure = cachingFailed)
```

Comment:
ToL: I'm not a fan of all these different ways of reaching `context stop self`.

I think it's more a hangover from how `BackendJobLifecycleActor` works though, so (shrug)